mappings/net/minecraft/util/math/WeightingUtil.mapping

```diff
@@ -0,0 +1,13 @@
+CLASS net/minecraft/class_6011 net/minecraft/util/math/WeightingUtil
+	METHOD method_34984 getTotalWeight (Ljava/util/List;)I
+		ARG 0 pool
+	METHOD method_34985 getRandomItem (Ljava/util/List;I)Ljava/util/Optional;
+		ARG 0 pool
+		ARG 1 totalWeight
+	METHOD method_34986 getRandomItem (Ljava/util/Random;Ljava/util/List;)Ljava/util/Optional;
```

Comment:
Though the string has `getRandomItem`, the name can be just `getRandom`, as the `Item` suffix is not descriptive and confuses with minecraft's inventory item.